core/domain/skill_services.py

```diff
@@ -378,6 +378,44 @@ def get_skill_descriptions_by_ids(topic_id, skill_ids):
     return skill_id_to_description_dict
 
 
+def get_rubrics_by_skill_ids(topic_id, skill_ids):
```

Comment:
Why `get_rubrics_by_skill_ids` has topic_id as a parameter?

Reply:
It is for the logging in the function. I used a similar structure as that was present in `get_skill_descriptions_by_ids`.

Reply:
If these methods are made just to fetch all the skills of a topic (not a random list of skill_ids), then I think it makes more sense to move this to topic_services.

Reply:
Yeah, done. Moved both into topic_fetchers.